topics/transcriptomics/tutorials/scrna_preprocessing/tutorial.md

```diff
@@ -0,0 +1,1374 @@
+
+The tutorial is structured in two parts:
+
+ ![Overview of workflow](../../images/scrna_workflow.svg "An overview of the entire workflow")
+
+The first part of this tutorial will use example *FASTQ* single batch data, which we will perform [barcode extraction](#understanding-barcodes) and annotation upon. Alignment and quality control will also be performed, and we will see how to construct a rudimentary count matrix. 
+
+The second part of this tutorial will deal with [multiple batches](#multiple-batches), and a different set of example count matrices will be used in which to merge and perform quality control upon. This will produce a final count matrix valid for downstream analysis.
+
+
+> ### Agenda
+>
+> In this tutorial, we will deal with:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Single Batch Processing
+
+### Data upload and organization
+
+In this tutorial we will be analysing scRNA-seq data of bone marrow cells taken from a single C57 mouse by *Herman et al.* ([2018](https://doi.org/10.1038/nmeth.4662)) and producing a count matrix that we can use for later analysis.
+
+The size of scRNA files (.fastq) are typically in the gigabyte range and are somewhat impractical for training purposes, so we will expediate the analysis by using a smaller subset of actual batch data. We will also be using *Mus Musculus* annotation data (.gtf) from the  [NCBI RefSeq](ftp://ftp.ncbi.nlm.nih.gov/genomes/refseq/vertebrate_mammalian/Mus_musculus/) track, as well as a barcodes file (.tsv).
+
+ ![Workflow Upper](../../images/scrna_workflow_upper.png "Single batch processing")
+
+
+> ### {% icon hands_on %} Hands-on: Data upload and organization
+>
+> 1. Create a new history and rename it (*e.g.* scRNA-seq single batch tutorial)
+> 1. Import the following files from [`Zenodo`](https://zenodo.org/record/2554612) or from the data library (ask your instructor)
+>    ```
+>    https://zenodo.org/record/2554612/files/SRR5683689_1.fastq.gz?download=1
+>    https://zenodo.org/record/2554612/files/SRR5683689_2.fastq.gz?download=1
+>    https://zenodo.org/record/2554612/files/Mus_musculus.GRCm38.93.mm10.UCSC.ncbiRefSeq.gtf?download=1
+>    https://zenodo.org/record/2554612/files/celseq_barcodes.192.tabular?download=1
+>    ``` 
+>    
+>    Set the database annotation to 'mm10' into the Genome dropdown box and select it.
+>    <img style='width:100%' src="../../images/scrna_select_data.png" />
+>    
+>    
+>    
+>    {% include snippets/import_via_link.md %}
+>    
+>
+> 1. Set the datatype of the `celseq_barcodes.192.tabular` to `tsv`
+> 1. Build a *Dataset pair* for the two FASTQ files
+>    - Click the *Operations on multiple datasets* check box at the top of the history panel
+>    - Check the two boxes next to the R1 and R2 scRNA FASTQ samples
+>    - Click *For all selected...* and choose *Build dataset pair*
+>    - Ensure that the forward read is the `_1` sample, and the reverse read is the `_2` sample.
+>      - Click 'Swap' otherwise.
```

Comment:
```suggestion
>      - Click **Swap** otherwise.
```

Reply:
The above suggestions are all examples where I think you should be more consistent with formatting:
- Text that appears in the tool interface (labels, select options, etc.) should be in **boldface**.
- Text that you are supposed to type into the tool interface literally should be enclosed in `backticks`.
  There are mutliple such issues throughout the document.

All the above unless @bebatut disagrees of course.